repository/t.json

```diff
@@ -2071,11 +2071,15 @@
 		{
 			"name": "Theme - Spacegray",
 			"details": "https://github.com/SublimeText/Spacegray",
-			"labels": ["theme"],
+			"labels": ["theme", "color scheme"],
 			"releases": [
 				{
-					"sublime_text": "*",
+					"sublime_text": ">=4000",
 					"tags": true
+				},
+				{
+					"sublime_text": "<4000",
+					"tags": "st3-"
```

Comment:
I may be blind, but I'm not seeing a prefixed release on the repo yet.

Reply:
Oh, maybe forgot to push.

Just did so, as well as a ST4-only 2.0.0 release.